### DISCUSSION TITLE
How do I upgrade to the latest version?

### DISCUSSION THREAD

**Discussion by bulletproof2k** [cross-seed/cross-seed#495] [Q&A] [answered]
I will be grateful if they tell me how to do it :]

**Comment by fabricionaweb** [comment]
That pretty much depends on how have you installed

**Reply by bulletproof2k** [reply]
Node, Yarn. I run it from the command line without going to the folder where the tag and the configuration file are located.

**Reply by fabricionaweb** [reply]
sorry to say but that doesnt helped much... 

have you installed using `yarn global add cross-seed`? is that what you're trying to say?

**Reply by mmgoodnow** [reply, marked as answer]
Just run it again

yarn global add cross-seed
